html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2><strong>by SnuggieStore</strong></h2>
+    <p>Price: <s>$19.99</s>$14.99</p>
+    <p>
+      Part of our new Americana collection, this <em>beautiful</em> blue is
+      already a customer favorite!
+    </p>
+    <p>Enjoy complete <b>Freedom of Movement!</b></p>
+    <p>
+      Does not slip and slide like a regular blanket and keeps you warm from
+      head to toe
```

Comment:
You would likely want `<strong>` here instead of `<b>` so that it also emphasizes for the screen readers/visual assistance